src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -22,6 +24,7 @@
 import jakarta.servlet.http.HttpServletRequest;
 import java.util.List;
 import java.util.UUID;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
can you use the terra exception here instead?
```suggestion
import bio.terra.common.exception.NotImplementedException;
```